src/perception/tag_detector/long_range_cam/long_range_tag_detector.processing.cpp

```diff
@@ -0,0 +1,214 @@
+#include "long_range_tag_detector.hpp"
+#include <cstdint>
+#include <opencv2/core/types.hpp>
+#include <sensor_msgs/Image.h>
+
+#include <opencv2/imgproc.hpp>
+
+namespace mrover {
+
+    /**
+     * Detects tags in an image, draws the detected markers onto the image, and publishes them to /long_range_tag_detection
+     *
+     * @param msg   image message
+     */
+    void LongRangeTagDetectorNodelet::imageCallback(sensor_msgs::ImageConstPtr const& msg) {
+        if (!mEnableDetections) return;
+
+        //Store image contents to member variables
+        updateImageMatrices(msg);
+
+        // 1. Detect tags
+        runTagDetection();
+
+        // 2. Update the hit counts of the tags in the mTags map
+        updateHitCounts();
```

Comment:
no hit counts on percep for long range tags

Reply:
could probably simplify a good amount of this pipeline by removing from struct + all modifications to it.

Reply:
we'd want to blow out and replace the entire tags data structure every iteration

Reply:
If there's no need to persist them over separate callbacks we could probably just publish as soon as we see them and have no data structure at all?